### PR TITLE
Fix local file location pathing

### PR DESCRIPTION
**Pull request by digitalsadhu**

### CODE REVIEW

File: lib/classes/local-file-location.js [hunk 1]
```diff
@@ -13,25 +13,30 @@ const mime = require('mime-types');
...
+        assert(!isAbsolute(path), '"path" must be a relative path');
+
         /**
          * @type {string} path relative path to file on disk
          */
         this.relative = path;
 
         /**
-         * @type {string} cwd current working directory
+         * @type {string} basePath to the file's relative location on disk
```
Comment (trygve-lie) [inline lib/classes/local-file-location.js:32]: Whats the difference between this and `this.relative`? Its possibly me finding the wording in this description to sound like  the description for `this.relative`.

File: lib/helpers/resolve-files.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+const ResolvedFiles = require('../classes/resolved-files');
+
+/**
+ * Create a new path from a path string preceeding a glob or the whole path if no glob is found
+ *
+ * @param {string} path
+ * @returns {string} modified path
+ */
+const pathUntilGlob = (path) => {
+    const segments = (path || '').split('/');
```
Comment (trygve-lie) [inline lib/helpers/resolve-files.js:21]: Will `.split('/');` work on windows? Should we be using [`path.sep`](https://nodejs.org/api/path.html#path_path_sep)?

File: lib/classes/local-file-location.js (hunk 1 above)
Comment (digitalsadhu) [reply lib/classes/local-file-location.js:32]: maybe bad wording on my part there, the absolute path is made up of the basePath + the relative path. So I meant to say there that that is the basePath to the files location on disk

File: lib/helpers/resolve-files.js (hunk 2 above)
Comment (digitalsadhu) [reply lib/helpers/resolve-files.js:21]: yeah should be path.sep indeed

**Review by trygve-lie — APPROVED** [review]
lgtm :+1:

**Pull request merged**